miner/bid_simulator.go

```diff
@@ -434,13 +430,11 @@ func (b *bidSimulator) newBidLoop() {
 				} else if !bestBidToRun.IsCommitted() {
 					// bestBidToRun is not committed yet, this newBid will trigger bestBidToRun to commit
 					bidRuntime = bestBidRuntime
-					replyErr = genDiscardedReply(bidRuntime)
```

Comment:
so if a bid have less reward,
will result `accepted` in line 484

suggested keep genDiscardedReply, but change it into 
```
	genDiscardedReply := func(betterBid *BidRuntime) error {
		return fmt.Errorf("bid was discarded for offering a lower reward than competing bids")
	}
```